mojo-executor-maven-plugin/src/main/java/org/twdata/maven/mojoexecutor/plugin/MojoExecutorMojo.java

```diff
@@ -83,13 +89,47 @@
      */
     private BuildPluginManager pluginManager;
 
-    public void execute() throws MojoExecutionException {
-        if ( mavenProject.getArtifactId().equals( "mojo-executor-test-project-null-maven-project" ))
+    public void execute() throws MojoExecutionException
+    {
```

Comment:
+1 , its simply because my IntelliJ env has that default; do you have a layout for Eclipse/IntelliJ I can apply?